style.css

```diff
@@ -0,0 +1,196 @@
+
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+.ul {
+  list-style: none;
+}
+
+.text-section {
+  background-image: url('img/Headerbg.jpg');
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 1rem;
+}
```

Comment:
- [OPTIONAL] You did an amazing job implementing the required changes requested by the previous reviewer :+1:, but I noticed that your background is not taking the whole screen size like the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), it has an empty space on each side. I strongly recommend you change the margin of the background to match the template :+1: 

**Your design:**

![Screenshot from 2023-01-13 06-19-45](https://user-images.githubusercontent.com/105117832/212308900-5055e050-ac98-4cda-bdf3-4a5010d33a6f.png)

**Figma design**

![Screenshot from 2023-01-13 06-20-27](https://user-images.githubusercontent.com/105117832/212308952-7c6fabcd-130c-4bef-8669-2b59f1cdaa5c.png)